functions.py

```diff
@@ -524,3 +585,18 @@ def xor(x, y):
 xor.__doc__ = _(
     'xor(x, y), logical xor. Returns True if either x is True \
 (and y is False) or y is True (and x is False), else returns False')
+
+
+def significant(x):
```

Comment:
See http://chemistry.bd.psu.edu/jircitano/sigfigs.html to see how to calculate significant digits

Reply:
I have incorporated the ideas in a new commit. Please see https://github.com/sugarlabs/calculate-activity/pull/61/commits/b016b2d78f403d23705f2812a84039783c9a3ff3